### PR TITLE
Big Data Madrid conferences

### PR DESCRIPTION
**Pull request by GomezIker**

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -143,3 +143,12 @@ tom_lomas:
...
     twitter: tomtheplank
     github: thomaslomas
+
+iker_gomez:
+    name: Iker Gomez
+    image: iker_gomez.jpg
+    role: Software Engineer
+    tribe: Data
+    startDate: 09-03-2015
+    bio: "Iker is one of our Software Engineers in the Data Team keeping our pipelines alive... He is the responsible of the data team improving their Spanish language skills"
```
Comment (WheresAlice) [inline _data/authors.yml]: responsible for, not responsible of

File: _posts/2016-03-25-big-data-madrid.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+date:       2016-03-25 10:00
+summary:    We sent Software Engineer Iker Gomez to Big Data Spain conferences in Madrid to learn more about big data technologies and real-time processing.
+category:   Big Data
+tags:       Big, Data, Spain, Madrid, Real-time, Predictive, Tech Ninja Fund
+author:     iker_gomez
+---
+
+<a name="english"></a>
+
+<a href="#spanish">[Llévame a la versión en castellano]</a>
```
Comment (WheresAlice) [inline _posts/2016-03-25-big-data-madrid.md]: Spanish or Castilian?

**Comment by cmbuckley** [comment]
Note to self for multilingual changes:

* https://github.com/jekyll/jekyll/issues/389
* http://stackoverflow.com/questions/22072495/multilingual-jekyll
* http://benoitpatra.com/2014/08/24/organize-a-multilanguage-jekyll-site/
* https://www.sylvaindurand.org/making-jekyll-multilingual/
* http://webdesign.tutsplus.com/articles/tips-for-designing-and-building-a-multilingual-website--cms-24708
* http://blog.codeinside.eu/2014/12/24/we-moved-completely-to-github-pages/

**Comment by WheresAlice** [comment]
:+1: LGTM

**Comment by ThomasLomas** [comment]
@GomezIker There's some merge conflicts with this post now. Assuming it's because of the change to `_data/authors.yml`

**Pull request merged**